### ISSUE TITLE
sort-imports: Inconsistent behaviour with named imports

### ISSUE THREAD

**Issue by last-partizan** [eslint/eslint#14570]
**What rule do you want to change?**
`sort-imports`

**Does this change cause the rule to produce more or fewer warnings?**
This change will produce less warnings, when modifying existing code.

**How will the change be implemented? (New option, new default behavior, etc.)?**
Default behaviour should be changed.

**Please provide some example code that this change will affect:**

This is `code 1`
```js
/* eslint no-unused-vars: "off" */
/* eslint sort-imports: "error" */
import { cloneDeep, debounce } from "lodash";
import { BForm } from "bootstrap-vue";
```

This is `code 2`
```js
/* eslint no-unused-vars: "off" */
/* eslint sort-imports: "error" */
import { cloneDeep, debounce } from "lodash";
import { BForm, BFormInput } from "bootstrap-vue";
```

**What does the rule currently do for this code?**
`code 1` is considered correct by eslint.
`code 2` raises error `Imports should be sorted alphabetically  sort-imports`

**What will the rule do after it's changed?**
Rule should either treat `single named import` as `multiple`, or it should change naming to `default` and `named` import groups to avoid confusion.

**Are you willing to submit a pull request to implement this change?**
No.

**Comment by nzakas** [comment]
Please edit your issue to use one of our issue templates:
https://github.com/eslint/eslint/tree/master/.github/ISSUE_TEMPLATE

We can take a look at that point. Thanks I’m advance.

**Comment by last-partizan** [comment]
@nzakas i updated issue to use `RULE_CHANGE.md` template.
